server/paymentProviders/stripe/creditcard.js

```diff
@@ -191,7 +191,8 @@ const createChargeAndTransactions = async (hostStripeAccount, { order, hostStrip
 
   // Create a Transaction
   const fees = extractFees(balanceTransaction);
```

Comment:
Just noting that extractFees is not super solid, let's say that for some reason there is a currency mismatch, we will be wrong. It would be better to read fees directly from the Stripe object:

```
 {
        "type": "stripe_fee",
        "amount": 332,
        "currency": "usd",
        "application": null,
        "description": "Stripe processing fees"
      },
```

This doesn't need to be addressed.

Reply:
@znarf : I'll leave this as it is for now since the only purpose of this `extractFees` object is to get the `stripeFees` and `applicationFees` for now; both of which we convert to proper values in place. 😄